christfellowshipchurch/web-app-v2
# 404 Update

### About
This PR implements changes to the 404 Page.

### Test Instructions
1. Go to [/404](https://www.christfellowship.church/404) on the live website and compare with [/404](https://web-app-v2-git-404-update-christ-fellowship-church.vercel.app/404) on the testing link.
2. 404 page on the testing link should look like the Figma design.
3. When clicking on search it should redirect to /discover with the text that was searched.
5. **ALL** links should redirect to expected pages.
6. Make sure to check mobile and iPad views as well.

### Screenshots
<img width="1440" alt="Screenshot 2024-03-07 at 11 43 37 AM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/cbb1f1d3-8339-4957-9ec2-bb38580bbd2a">

### Closes Tickets
[CFDP-2894](https://christfellowshipchurch.atlassian.net/browse/CFDP-2894)

[CFDP-2894]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2894?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
